### PR TITLE
feat(*): allow styling for shadow and border - I124

### PR DESCRIPTION
**Pull request by PseudoNerd**
# Issue #124 
<OVERALL SUMMARY OF PULL REQUEST>
Added additional props for Shadow and border attributes

### Changes
-  Added props for code flexibility
  - Additional props for Border and Box-Shadow attributes of Markdown Editor
- Changed src/SlateAsInputEditor/index.js

### Flags
- <DESCRIBE ISSUES OR HOLDS FOR REVIEWERS>

### Related Issues
- Issue #<NUMBER HERE>
- Pull Request #<NUMBER HERE>

### CODE REVIEW

**Comment by PseudoNerd** [comment]
> I think we should also update instructions in the `README`

Modified the README file and updated the PR.

File: src/SlateAsInputEditor/index.js [hunk 1]
```diff
@@ -36,8 +36,8 @@ const EditorWrapper = styled.div`
   min-height: 750px;
   max-width: ${props => props.width || 'none'};
   min-width: ${props => props.width || 'none'};
-  border: 1px solid #979797;
-  border-radius: 10px;
+  border: ${props => props.EDITOR_BORDER || ' 1px solid #979797'};
+  box-shadow: ${props => props.EDITOR_SHADOW || ' 1px 2px 4px rgba(0, 0, 0, .5)'};
```
Comment (irmerk) [inline src/SlateAsInputEditor/index.js]: I think we still want `border-radius`, to optionally make it `10px` for example. @michael-grover @DianaLease thoughts?

Comment (Michael-Grover) [reply src/SlateAsInputEditor/index.js]: yeah, different applications of this editor have different border radiuses

Comment (PseudoNerd) [reply src/SlateAsInputEditor/index.js]: Should I go ahead and add an optional border-radius for the editor?

Comment (irmerk) [reply src/SlateAsInputEditor/index.js]: @pseudonerd yes please!

Comment (PseudoNerd) [reply src/SlateAsInputEditor/index.js]: I have updated the PR.

File: README.md [hunk 2]
```diff
@@ -114,6 +114,8 @@ This is an object with the following possible css inputs as strings:
 - `BUTTON_SYMBOL_ACTIVE`
 - `DROPDOWN_COLOR`
 - `TOOLBAR_BACKGROUND`
+- `EDITOR_SHADOW`
+- `EDITOR_BORDER`
```
Comment (irmerk) [inline README.md]: Just add `EDITOR_BORDER_RADIUS` here

Comment (PseudoNerd) [reply README.md]: Updated the README file.

**Pull request merged**